eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
 }
 
 describe("Destination Picker Menu", () => {
-  it("should default to current section tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText("Question one")).toBeVisible();
-  });
+  describe("Hub is disabled", () => {
+    beforeEach(() => {
+      questionnaire = { hub: false };
+      useQuestionnaire.mockImplementation(() => ({ questionnaire }));
+    });
 
-  it("should show active styling for current tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should default to current section tab", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText("Question one")).toBeVisible();
+    });
```

Comment:
I'm a little confused as to what this is testing. Could you rename it to make it clearer, please?